src/Meshes/abstractMesh.ts

```diff
@@ -1099,14 +1099,43 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
             return this._masterMesh.getBoundingInfo();
         }
 
-        if (!this._boundingInfo) {
-            // this._boundingInfo is being created here
+        if (!this._boundingInfo || this._boundingInfoIsDirty) {
```

Comment:
would it work if you default _boundingInfoIsDirty to true so that you only need to check the flag?

Reply:
good idea